notifique/src/main/java/com/nathanrassi/notifique/NotificationListener.kt

```diff
@@ -0,0 +1,11 @@
+package com.nathanrassi.notifique
+
+import android.app.Service
+import android.content.Intent
+import android.os.IBinder
+
+class NotificationListener() : Service() {
+    override fun onBind(p0: Intent?): IBinder {
```

Comment:
this parameter is not nullable.
also, this parameter should use the super's the parameter name: `intent`.